.gitignore

```diff
@@ -0,0 +1,8 @@
+### Composer template
+composer.phar
+/vendor/
+.php-cs-fixer.cache
+
```

Comment:
Dodałbym gitignore od phpstorma, czyli albo https://www.toptal.com/developers/gitignore/api/phpstorm, albo `.idea/*`.

Reply:
Myślę, że właściwszym miejscem na trzymanie ignorów do IDE jest lokalny profil użytkownika w systemie gdzie można sobie zdefiniować globalnego gitignore'a. Chyba nie powinniśmy zakładać, że wszyscy użyją tego samego IDE do edycji a pchanie tam ignorów dla wszystkich IDE raczej mija się z celem.

Patrz np. tu: 
https://medium.com/self-modifying-code/create-a-global-gitignore-step-by-step-for-macos-and-windows-31a765291409